src/xrc/src/periodic.rs

```diff
@@ -536,5 +536,15 @@ mod test {
             forexes_with_timestamps_and_context[5].2.timestamp,
             1680134400
         );
+
+        assert!(matches!(
+            forexes_with_timestamps_and_context[6].0,
+            Forex::ReserveBankOfAustralia(_)
+        ));
+        assert_eq!(forexes_with_timestamps_and_context[5].1, 1680134400);
+        assert_eq!(
+            forexes_with_timestamps_and_context[5].2.timestamp,
```

Comment:
Should these be:
`forexes_with_timestamps_and_context[6].1` and `forexes_with_timestamps_and_context[6].2`

Reply:
Good catch. Fixed.